src/Paket.Core/Common/Constants.fs

```diff
@@ -24,7 +24,7 @@ let [<Literal>] BootstrapperFileName      = "paket.bootstrapper.exe"
 let [<Literal>] PaketFileName             = "paket.exe"
 let [<Literal>] TargetsFileName           = "paket.targets"
 let [<Literal>] ReferencesFile            = "paket.references"
-let [<Literal>] AccessLockFileName        = "paket.locked"
+let [<Literal>] AccessLockFileName        = ".processlock"
```

Comment:
I think it should be `paket.processlock` - just in case people have gitignore rules like `paket.*` (even if this file is not really important for git)

Reply:
don't worry. will do that right now